### PR TITLE
Single Page Application Pull Request

### PR DESCRIPTION
**Pull request by adamteddychang**
Features implemented : 
- Added navbar
- Added contact section
- Added ability to display different sections after clicking different nav-items
- In collaboration with @Chimwemwe-127001

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,34 +8,67 @@
...
+  </footer>
   <script src="script.js"></script>
+  <script src="test.js"></script>
   <script>
     const removeBook = (bookId) => Books.remove(bookId);
+    const { DateTime } = luxon;
+    const localTime = DateTime.local();
+    const now = localTime;
+    document.getElementById('date-time').innerHTML = now;
   </script>
```
Comment (eliasecasta) [inline index.html]: - [x] Please keep all the JavaScript code in JS files, not in the script tag to comply with JS best practices.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @adamteddychang @Chimwemwe-127001, I am Elias, your reviewer for this PR

Well done so far with your project!

I really liked the trippy design that you used for this project, everything is working as expected, great job!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] Please remove all JS code from your HTML file in order to comply with JS best practices.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (adamteddychang) [reply index.html]: We've done as requested. Thank you for the review.

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇 
#### Hello team, 
Nice work! Your project is complete!  You have met all requirements for this project. It is a thumbs up for me. 👍🏽  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**